lib/sync-DataSetModel.js

```diff
@@ -49,6 +51,7 @@ var self = {
   },
 
   doListHandler: function (dataset_id, query_params, cb, meta_data) {
+   
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)